client/src/components/Bills/BillForm.js

```diff
@@ -0,0 +1,194 @@
+import { useState } from "react";
+import ContactDropDown from "../DropDowns/ContactDropList";
+
+const emptyBill = {
```

Comment:
💡 Maybe add an `emptyBill = Object.freeze({ ....} )`?

Reply:
I want the values in the emptyBill to change as the user inputs information into the form. Wouldn't Object.freeze() prevent that from happening?